### PR TITLE
Pepper-422 Pepper-423 os2 germline consent addendum v3

### PR DESCRIPTION
**Pull request by ssettipalli**
Pepper-422 Pepper-423 os2 germline consent addendum activities v3

## FUD Score

_Overall, how are you feeling about these changes?_

- [x ] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [x ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [ x] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/osteo/Osteo2GermlinePedConsentVersion3.java [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+        this.jdbiRevision = handle.attach(JdbiRevision.class);
+
+        log.info("Revisioning: {} ", activityCode);
+        ActivityVersionDto version3ForConsent = getVersion(handle, studyDto, metaConsent, activityCode, versionTag);
+
+        runGermlinePedConsentUpdate(handle, metaConsent, version3ForConsent, dataCfg);
+
+        //TODO
+        //disable parent/guardian Date Question & add new Content block with sys date
+        disableQuestionAndAddContent(handle, studyDto.getId(), metaConsent, version3ForConsent, DATE_QUESTION_STABLEID_PAR, BLOCK_ADD_PAR);
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/osteo/Osteo2GermlinePedConsentVersion3.java:121]: Date was incorrectly coded as user input where it should be a system date.
corrected in this revision
disabled the input Date question and added as a system date

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/osteo/Osteo2GermlinePedConsentVersion3.java [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+        //disable Child Date Question & add new Content block with sys date
+        disableQuestionAndAddContent(handle, studyDto.getId(), metaConsent, version3ForConsent, DATE_QUESTION_STABLEID_CHD, BLOCK_ADD_CHD);
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/osteo/Osteo2GermlinePedConsentVersion3.java:124]: Date was incorrectly coded as user input where it should be a system date.
corrected in this revision
disabled the input Date question and added as a system date

File: pepper-apis/studybuilder-cli/studies/osteo/patches/germline-consent-addendum-ped-v3.conf [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            ]
+          },
+          "blockType": "CONTENT",
+          "shownExpr": null
+        }
+      "old_template_search_text": "$study_contact_pediatric</p>"
+    }
+
+  ],
+  "block-add-parent":
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/patches/germline-consent-addendum-ped-v3.conf:87]: Date was incorrectly coded as user input where it should be a system date.
corrected in this revision
disabled the input Date question and added as a system date

File: pepper-apis/studybuilder-cli/studies/osteo/patches/germline-consent-addendum-ped-v3.conf [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+      "bodyTemplate": {
+        "templateType": "HTML",
+        "templateText": """<p class="paragraph">$ddp.date("MM / dd / yyyy")</p>""",
+        "variables": []
+      },
+      "blockType": "CONTENT",
+      "shownExpr": """user.studies["cmi-osteo"].forms["GERMLINE_CONSENT_ADDENDUM_PEDIATRIC"].questions["ADDENDUM_CONSENT_BOOL_PEDIATRIC"].answers.hasTrue()"""
+    }
+
+  "block-add-child":
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/patches/germline-consent-addendum-ped-v3.conf:110]: Date was incorrectly coded as user input where it should be a system date.
corrected in this revision
disabled the input Date question and added as a system date

**Review by otchet-broad — APPROVED** [review]
LGTM!  Thanks for providing example users I could walk through to see the changes on dev!

**Pull request merged**